webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/DataGridContextMenu/DataGridContextMenuCellEditingService.ts

```diff
@@ -94,12 +97,12 @@ export class DataGridContextMenuCellEditingService {
         const editor = model.source.getAction(resultIndex, IDatabaseDataEditAction);
         const select = model.source.tryGetAction(resultIndex, IDatabaseDataSelectAction);
 
-        const cellValue = view.getCellValue(key);
+        const cellHolder = view.getCellValue(key);
```

Comment:
Do we still want to call this method getCellValue even though it's not returning a value anymore?